shanoir-ng-datasets/src/main/java/org/shanoir/ng/solr/service/SolrServiceImpl.java

```diff
@@ -101,25 +115,73 @@ public void deleteFromIndex(List<Long> datasetIds) throws SolrServerException, I
 	public void deleteAll() throws SolrServerException, IOException {
 		solrJWrapper.deleteAll();
 	}
-	
-	@Transactional
```

Comment:
I'm not sure that it still works if this method is not transactional anymore, as shanoirMetadataRepository.findAllAsSolrDoc(); accesses the database.
As said earlier, maybe merge both methods ?